package.json

```diff
@@ -37,6 +37,7 @@
     "eslint-plugin-flowtype": "^2.39.1",
     "flow-bin": "^0.66.0",
     "kocha": "^1.8.1",
+    "nyc": "^11.6.0",
```

Comment:
This root package should also have this module. See https://github.com/phenyl-js/phenyl/blob/master/tools/lib/phenyl-module.js#L134.
`npm run load` will automatically make symlinks of nyc in each node_modules directory.

The only thing we should write is to link the executable file in `.bin` like here. https://github.com/phenyl-js/phenyl/blob/master/tools/lib/phenyl-module.js#L210